spec/features/users/piv_cac_management_spec.rb

```diff
@@ -11,123 +11,115 @@ def find_form(page, attributes)
     let(:uuid) { SecureRandom.uuid }
     let(:user) { create(:user, :signed_up, :with_phone, with: { phone: '+1 202-555-1212' }) }
 
-    context 'with an account allowed to use piv/cac' do
```

Comment:
The only changes in this file are I removed this context which no longer applies and the before action below it which also no longer applies.

For context, we use to only allow people with government emails from certain agencies to use PIV or CAC. We have removed that requirement and allow anyone to set up with a PIV or CAC now.